### ISSUE TITLE
VS 2019 hangs if class declaration specifies inheritance from itself (typo)

### ISSUE THREAD

**Issue by leon-ts** [X-Sharp/XSharpPublic#1032]
There is the following example:
```
CLASS TestClass INHERIT TestClass

    METHOD DoSomething() AS VOID STRICT
         RETURN

END CLASS
```

Hovering over the word **TestClass** causes VS 2019 to hang.

VS 2019: 16.11.14
X# 2.9.1.1

**Comment by VolkmarR** [comment]
@leon-ts have you tried to update to x# 2.11.1 version? That should fix this issue.

**Comment by leon-ts** [comment]
@VolkmarR 
My applications are already in production on version 2.9.1.1. Therefore, I am in no hurry to update X# to the latest versions. But I will update to version 2.12 after its release. The problem with the hang is not critical, just a typo.

**Comment by RobertvanderHulst** [comment]
Leonid,
If you make a backup copy of the X# 2.9.1.1 runtime, and install a later build, then you can replace the DLLs in the GAC with the 2.9.1.1 DLLs and still compile against the 2.9.1.1 runtime. You would then be using the 2.11 (or 2.12) compiler and VS Integration but the 2.9 runtime.
We are planning to release the runtime and compiler/tools separately in the not too far future (the runtime will then also become available as nuget packages).

**Comment by RobertvanderHulst** [comment]
This is already fixed in a build after 2.9
